app/src/main/res/values/strings.xml

```diff
@@ -119,4 +119,9 @@
 
     <!-- KeyManagementActivity -->
     <string name="pref_key_management_header">Key management</string>
+    <string name="create_proflie_too_short_password">The password is too short, it should contain at least 6 characters</string>
+    <string name="create_profile_empty_password">Password shouldn\'t be empty</string>
```

Comment:
"You must select a password" (I don't think using the word "empty" is nice here)